elastic/kibana
# [Fleet] add readme to uploaded package info and fix images

#84583

- Add `readme` property and path if exists to package info
- Have images in package details page use the `src` over the `path` property

Test
- Upload a package
- Visit Package details page of the uploaded package.  See readme.md, screenshots, and logo loading correctly.

[screenshot coming]
